message_tagging_service/messaging.py

```diff
@@ -23,6 +23,7 @@
 import json
 
 from message_tagging_service import conf
+from message_tagging_service import monitor
```

Comment:
why don't use a one-line import?
Example: from message_tagging_service import conf, monitor
The same also above.

Reply:
Yeah, it's ok to use one-line import here. :)

Reply:
@gnaponie Patch is updated to address your comment. Thanks. :)